src/bindings/jna/libelektra/src/main/java/org/libelektra/exception/KeyBinaryValueException.java

```diff
@@ -0,0 +1,13 @@
+package org.libelektra.exception;
+
+import org.libelektra.Key;
+
+/**
+ * Indicates a {@link Key key's} underlying native key value is not of type
```

Comment:
```suggestion
 * Indicates a {@link Key key's} underlying native key value is not
```

Reply:
I would rather leave "of type binary"... as opposed to "of type string" in `KeyStringValueException`... IMHO the value of a key being binary means only being able to assume 2 distinctive states.